pkg/watcher/internal/metricsprovider/prometheus.go

```diff
@@ -33,13 +43,15 @@ import (
 )
 
 const (
-	DefaultPromAddress = "http://prometheus-k8s:9090"
-	promStd            = "stddev_over_time"
-	promAvg            = "avg_over_time"
-	promCpuMetric      = "instance:node_cpu:ratio"
-	promMemMetric      = "instance:node_memory_utilisation:ratio"
-	allHosts           = "all"
-	hostMetricKey      = "instance"
+	EnableOpenShiftAuth = "EnableOpenShiftAuth"
```

Comment:
Can we follow CAPITAL_LETTER_CONSTANT format like "METRICS_PROVIDER_ADDRESS", "METRICS_PROVIDER_NAME"?